examples/jenkins/application-template.json

```diff
@@ -314,7 +349,27 @@
     }
   ],
   "parameters": [
+     {
+      "name": "DEPLOYMENT_REQUEST_SIZE",
+      "description": "Memory rquested on startup from OpenShift Quota System",
+      "value": "64Mi"
+    },
     {
+      "name": "DEPLOYMENT_LIMIT_SIZE",
+      "description": "Maximum amount of memory allowed by OpenShift Quota System",
+      "value": "128Mi"
+    },
+     {
+      "name": "BUILD_REQUEST_SIZE",
+      "description": "Memory rquested on startup from OpenShift Quota System",
+      "value": "32Mi"
+    },
+    {
+      "name": "BUILD_LIMIT_SIZE",
```

Comment:
In my testing for the online beta, where a quota was defined, the pod would not be started (with an explicit message stating such) unless a limit or request was defined.  That coupled with the idea of updating the origin templates to service the beta given to me lead to his PR.

I'll switch the build resource element to a request.

I'm assuming that BUILD_LIMIT_SIZE is not liked as well.  I'll go with "MEMORY_REQUEST"  (in line with MEMORY_LIMIT, unless that is not liked), update the description with the implications to the user pulled from the doc, etc.